cmd/prometheus/main_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/prometheus/prometheus/util/testutil"
 )
 
+var cwd string
```

Comment:
Why is this a global?

Reply:
It is computed in test_main.

I can rescope it in my test.